helix-view/src/document.rs

```diff
@@ -50,6 +51,7 @@ pub struct Document {
 
     diagnostics: Vec<Diagnostic>,
     language_server: Option<Arc<helix_lsp::Client>>,
+    line_ending: Option<LineEnding>,
```

Comment:
Doesn't have to be an option because we always expect to have a line ending.

Reply:
auto_detect_line_ending can return None in some instances.

Reply:
Ah I see, we just set the default one then.